docs/getting-started-guides/libvirt-coreos.md

```diff
@@ -117,6 +117,26 @@ setfacl -m g:kvm:--x ~
 
 By default, the libvirt-coreos setup will create a single Kubernetes master and 3 Kubernetes nodes. Because the VM drives use Copy-on-Write and because of memory ballooning and KSM, there is a lot of resource over-allocation.
 
+There are two major ways of how-to run and configure the libvirt-based Kubernetes cluster on CoreOS instances:
+
+#### Automated setup
+
+To start and setup your local CoreOS-Kubernetes cluster the following command might be easily runned:
+
+`export KUBERNETES_PROVIDER=libvirt-coreos; wget -q -O - https://get.k8s.io | bash`
+
+or, in favor of curl:
+
+`export KUBERNETES_PROVIDER=libvirt-coreos; curl -sS https://get.k8s.io | bash`
+
+Just wait a few minutes, the tarball with the latest Kubernetes release will be downloaded; CoreOS instances will be spawned and Kubernetes cluster on them will be launched.
+
+Cluster configuration will be the following: 4 KVM/QEMU instances, where 1 instance will act as a Kubernetes Master node and 3 as Minion nodes.
+
+If you'd like to run this cluster with customized settings, run it in a manual way:
```

Comment:
If you'd like to run this cluster with customized settings, follow the [manual setup instructions](#manual-setup).